www/main/navigator/search/components/SearchContent.jsx

```diff
@@ -99,15 +99,21 @@ const SearchContent = () => {
     setDatabaseProgress(data.percent);
   });
 
+  let placeholder = '';
+
+  if (databaseProgress < 1) {
+    placeholder = i18n.t('DATABASE.DOWNLOADING');
+  } else if (currentSearchType === 3) {
+    placeholder = i18n.t('SEARCH.PLACEHOLDER_ENGLISH');
+  } else {
+    placeholder = i18n.t('SEARCH.PLACEHOLDER_GURMUKHI');
+  }
+
```

Comment:
I think it should go to a function rather than a direct if statement.

Reply:
Done @theamanjs ji, have a look.